1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,118 @@
+
+**Note:** Only Linux and OS X are supported currently.
+
+##  SOCKS
+Run the following command from the DC/OS CLI:
+
+```
+$ dcos tunnel socks
+```
+
+Configure your application to use the proxy on port 1080: `127.0.0.1:1080`
+
+### Port Forwarding
+The SOCKS proxy works by port forwarding. Append `.mydcos.directory` to the end of your domain when you enter commands. For instance, `http://example.com:1080/?query=hello` becomes `http://example.com.mydcos.directory:1080/?query=hello`.
+
+##  HTTP
+Run the following command from the DC/OS CLI:
+
+```
+$ sudo dcos tunnel http
+```
+
+To run the HTTP proxy without root privileges, use the `--port` flag to configure it to use another port:
+
+```
+$ dcos tunnel http --port 8000
+```
+
+Then, configure your application to run HTTP on the port you specified above.
+
+### Port Forwarding
+The HTTP proxy works by port forwarding. Append `.mydcos.directory` to the end of your domain when you enter commands. For instance, `http://example.com:8080/?query=hello` becomes `http://example.com.mydcos.directory:8080/?query=hello`.
+
+### SRV Records
+The HTTP proxy exposes DC/OS SRV records as URLs in the form `_<port-name>._<service-name>._tcp.marathon.mesos.mydcos.directory`.
+
+To name a port, add `name` to the `portMappings` or `portDefinitions` field of a Marathon application definition. Whether you use `portMappings` or `portDefinitions` depends on whether you are using `BRIDGE` or `HOST` networking. [Learn more about networking and ports in Marathon](https://mesosphere.github.io/marathon/docs/ports.html).
+
+```json
+"portMappings": [
+    {
+        "name": "<my-port-name>",
+        "containerPort": 3000,
+        "hostPort": 0,
+        "servicePort": 10000,
+        "labels": {
+             "VIP_0": "1.1.1.1:30000"
+        }
+    }
+]
+```
+
+```json
+"portDefinitions": [
+    {
+      "name": "<my-port-name>",
+      "protocol": "tcp",
+      "port": 0,    
+    }
+  ]
+```
+
+Alternatively, you can add a named port from the DC/OS web interface. Go to the **Services** tab, click the name of your service, and then click **Edit**. Enter a name for your port on the **Network** tab.
+
+The `<service-name>` is the value of the `id` field in your Marathon application definition or the entry in the **ID** field of a service you create from the DC/OS web interface.
+
+## VPN
+Run the following command from the DC/OS CLI
+
+```
+$ sudo dcos tunnel vpn
+```
+
+The VPN client attempts to auto-configure DNS, but this functionality does not work on Mac OSX. To use the VPN client on OSX, [add the DNS servers](https://support.apple.com/kb/PH18499?locale=en_US) that DC/OS Tunnel instructs you to use.
+
+The VPN does not work by port forwarding, so you do not need to append `.mydcos.directory` to the end of your commands. For example, to access the DC/OS web interface from within the cluster, point your browser to `https://master.mesos`.
```

Comment:
Maybe we can remove this now that adding `.mydcos.directory` is a minority and only related to the HTTP proxy.